doc/support_status.md

```diff
@@ -127,55 +127,65 @@ ______
 
 | Tensorflow Op        | Supported ONNX Version  |
 | -------------- |:------------------:|
-|add|1, 6|
-|arg_max|1|
-|arg_min|1|
-|avg_pool|1, 7|
-|bias_add|1, 6|
-|cast|1|
-|ceil|1, 6|
-|concat_v2|1, 4|
-|conv1_d|1|
-|conv2_d|1|
-|conv3_d|1|
-|equal|1|
-|exp|1, 6|
-|fill|1|
-|floor|1, 6|
-|fused_batch_norm|1, 6|
-|greater|1|
-|identity|1|
-|less|1|
-|log|1, 6|
-|log_softmax|1|
-|logical_and|1, 6|
-|logical_not|1|
-|logical_or|1|
-|logical_xor|1|
-|mat_mul|1|
-|max|1|
-|max_pool|1|
-|maximum|1|
-|mean|1|
-|min|1|
-|mul|1, 6|
-|pad|1, 2|
-|pow|1|
-|prod|1|
-|random_standard_normal|1|
-|random_uniform|1|
-|real_div|1, 6|
-|reciprocal|1, 6|
-|relu|1, 6|
-|reshape|1, 5|
-|shape|1|
-|sigmoid|1, 6|
-|softmax|1|
-|split_v|1, 2|
-|sqrt|1, 6|
-|squeeze|1|
-|sub|1, 6|
-|sum|1|
-|tanh|1, 6|
-|tile|1|
-|transpose|1|
+|Abs|1, 6|
```

Comment:
The header for this column is Tensorflow ops, looks lie something has changed? Maybe you are listing supported ONNX operators now?

Reply:
It's tensorflow ops just before applying name_to_lower.